### PR TITLE
Add `fluent-plugin-opensearch`

### PR DESCRIPTION
**Pull request by r2bit**
This new official plugin replaces the `aws-elasticsearch-plugin`.
It provides SigV4 signing and much more, e.g. 400 status debugging.

https://github.com/fluent/fluent-plugin-opensearch

### CODE REVIEW

**Comment by r2bit** [comment]
!build

**Comment by r2bit** [comment]
!build

**Comment by sm-deployer** [comment]
Built and published `1.14.4-196aef6`

**Comment by sm-deployer** [comment]
Built and published `1.14.4-196aef6`

File: Gemfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ gem "fluent-config-regexp-type", "1.0.0"
 gem "fluent-logger", "0.9.0"
 gem "fluent-plugin-aws-elasticsearch-service", "2.4.1"
```
Comment (mpi1) [inline Gemfile:5]: I guess the `fluent-plugin-aws-elasticsearch-service` could be removed, as it doesn't seem to be [dependency](https://github.com/fluent/fluent-plugin-opensearch/blob/main/fluent-plugin-opensearch.gemspec#L25-L29) for `fluent-plugin-opensearch`.

Comment (mpi1) [reply Gemfile:5]: Also, I think `fluent-plugin-td-monitoring` could be removed. Link to it's [GH](https://github.com/treasure-data/fluent-plugin-td-monitoring)

**Review by mpi1 — APPROVED** [review]

**Pull request merged**